pages/chapter/[chapterNumber].js

```diff
@@ -151,11 +153,19 @@ export default function Chapter({ chapterData }) {
             <button
               type="button"
               className="-ml-px relative inline-flex items-center space-x-2 px-4 py-2 border border-gray-300 text-sm font-medium dark:bg-dark-100 rounded-r-md text-gray-700 dark:text-gray-50 bg-gray-50 hover:bg-gray-100 dark:hover:bg-dark-bg focus:outline-none focus:ring-1 focus:ring-my-orange focus:border-my-orange"
+              onClick={() => setisAscSorted(!isAscSorted)}
             >
-              <SortAscendingIcon
-                className="h-5 w-5 text-gray-400 dark:text-gray-50"
-                aria-hidden="true"
-              />
+              {isAscSorted ? (
+                <SortDescendingIcon
+                  className="h-5 w-5 text-gray-400 dark:text-gray-50"
+                  aria-hidden="true"
+                />
+              ) : (
+                <SortAscendingIcon
+                  className="h-5 w-5 text-gray-400 dark:text-gray-50"
+                  aria-hidden="true"
+                />
+              )}
               <span>Sort</span>
 
               <ChevronDownIcon
```

Comment:
Should we remove this icon? what do you think?

Reply:
So if everything seems fine , you can merge?

Reply:
Sure, we will iterate over it in the future. Again thanks @Aniket-508